nifi-toolkit/nifi-toolkit-tls/src/main/java/org/apache/nifi/toolkit/tls/standalone/TlsToolkitStandaloneCommandLine.java

```diff
@@ -122,12 +129,25 @@ public static void main(String[] args) {
         } catch (CommandLineParseException e) {
             System.exit(e.getExitCode().ordinal());
         }
-        try {
-            new TlsToolkitStandalone().createNifiKeystoresAndTrustStores(tlsToolkitStandaloneCommandLine.createConfig());
-        } catch (Exception e) {
-            tlsToolkitStandaloneCommandLine.printUsage("Error generating TLS configuration. (" + e.getMessage() + ")");
-            System.exit(ExitCode.ERROR_GENERATING_CONFIG.ordinal());
+
+        StandaloneConfig conf = tlsToolkitStandaloneCommandLine.createSplitKeystoreConfig();
```

Comment:
I think it makes sense to check `isSplitKeystore` before creating this config object because currently this is always created whether it is needed or not.

Reply:
Updated this to use the command line parsed args rather than the config, and will no longer create the config unless it's required.